src/jp2_remediator/processor.py

```diff
@@ -22,34 +24,36 @@ def process_directory(self, directory_path):
             for file in files:
                 if file.lower().endswith(".jp2"):
                     file_path = os.path.join(root, file)
-                    print(f"Processing file: {file_path}")
+                    self.logger.info(f"Processing file: {file_path}")
```

Comment:
Since this code (lines 27-29) are identical to the `process_file(...)` method, please replace this code with a call to `self.process_file(...)`.